src/Adapter/Requests/Services/DefaultRequestInvoker.cs

```
@@ -120,9 +115,8 @@ private async Task TrySaveBucket(Bucket? bucket, CancellationToken cancellationT
                     await bucketRepository.Save(bucket, cancellationToken);
                 }
             }
-            catch (Exception exception)
+            catch (Exception)
```

Add back exception logging here

---

Actually going to let exceptions propagate up to the invoker here, so that the transaction can be rolled back if there was an issue.